flocker/provision/_install.py

```diff
@@ -122,6 +129,17 @@ def task_upgrade_kernel():
     ]
 
 
+def task_upgrade_kernel_centos():
+    return [
+        Run.from_args([
+            "yum", "install", "-y", "kernel-devel", "kernel"]),
+        # For dkms and ... ?
+        Run.from_args([
+            "yum", "install", "-y", "epel-release"]),
```

Comment:
The fact that CentOS includes the `epel-release` package in its repositories suggests that many people on CentOS do install EPEL as well. Since EPEL specifically doesn't replace upstream packages, installing this only allows you to install packages you wouldn't otherwise be able to install.